bin/clover/src/pipeline-steps/assetSpecificOverrides.ts

```diff
@@ -241,6 +241,17 @@ const overrides = new Map<string, OverrideFn>([
     setAnnotationOnSocket(groupSocket, { tokens: ["GroupId"] });
     variant.sockets.push(groupSocket);
   }],
+  ["TargetGroup Targets", (spec: ExpandedPkgSpec) => {
+    const variant = spec.schemas[0].variants[0];
+
+    // Add an annotation for the Id output socket to connect to HostedZoneId
+    const socket = variant.sockets.find(
+      (s: ExpandedSocketSpec) => s.name === "Id" && s.data.kind === "input",
+    );
+    if (!socket) return;
+
+    setAnnotationOnSocket(socket, { tokens: ["InstanceId"] });
```

Comment:
Should we add annotations for other target types?

Reply:
Good shout @sprutton1 - we should add ARN as well - as it will allow [other] albs and lambda targets - @adamhjk can you update>

Reply:
done!